docs/simple_switch.md

```diff
@@ -94,6 +214,8 @@ queue.
 - `qid`: when there are multiple queues servicing each egress port (e.g. when
 priority queueing is enabled), each queue is assigned a fixed unique id, which
 is written to this field. Otherwise, this field is set to 0.
+TBD: `qid` is not currently part of type `standard_metadata_t` in v1model.
+Perhaps it should be added?
```

Comment:
@antoninbas Should I maybe create a separate issue to add the `qid` field to v1model.p4 include file?